.github/workflows/run-tests-linux-multiarch.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       matrix:
         arch: [linux-arm64, linux-x86]
         scala: [2.13.10, 3.2.1]
-        build-mode: [debug, release-fast]
+        build-mode: [debug, release-fast, release-size]
```

Comment:
Whenever possible let's limit the new mode to single included tests. 
Our current CI is already taking a very long time to finish and we would need to refactor it soon to reduce CI times.
A single entry in `include` should do the job,  especially since in most of the optimizer combinations would be already  in debug in release-fast

Reply:
thanks, I've updated CI config, can you please review again?